marlowe/doc/embedded-marlowe.adoc

```diff
@@ -1,5 +1,5 @@
 [#embedded-marlowe]
-== Embedded Marlowe
+== Marlowe embedded in Haskell
```

Comment:
`Marlowe embedded in Haskell` but `Marlowe in Javascript`

Reply:
Done.

Reply:
Done